packages/node/src/example-import.ts

```diff
@@ -1,14 +0,0 @@
-// We intentionally import these types here
-// which will fail at compile time if exports
-// are not found in the index file
```

Comment:
This file was not actually testing the build output so I deleted in.

Instead, we'll rely on the existing `test/fixtures/15-helpers` and make sure the build output is copied as the final step in the build before tests begin.